mira/dkg/construct.py

```diff
@@ -183,8 +190,10 @@ def load():
     click.secho(command, fg="blue")
     os.system(command)  # noqa:S605
 
-    time.sleep(10)
-    os.system("brew services restart neo4j")
+    if restart:
+        time.sleep(10)
+        click.secho("restarting neo4j...")
+        os.system("brew services restart neo4j")
```

Comment:
Using `brew` will not work generally. Both on Mac and Linux, however, the `neo4j restart` command works as expected, at least for me so perhaps we should use that here.

Reply:
Subsumed by https://github.com/indralab/mira/pull/13#issuecomment-1218746374